src/jobs/resync_repositories_job.cr

```diff
@@ -9,14 +9,12 @@ class ResyncGithubRepositoriesJob < PeriodicJobWithErrorHandler
     repositories =
       Repository
         .query
-        .where({provider: "github"})
+        .where{(synced_at == nil) | (synced_at > 1.day.ago) }
```

Comment:
In particular, I have not validated that this query produces the expected SQL. I have no experience with the Clear ORM and the documentation around this particular query need is a little lacking.

Reply:
This isn't necessary since we always fetch repositories ordered by the last `synced_at`. And all existing repositories already have this.